changelogs/fragments/1305-added-xfconf-tests.yaml

```
@@ -0,0 +1,4 @@
+minor_changes:
+  - xfconf - added unit tests (https://github.com/ansible-collections/community.general/issues/695).
+  - xfconf - removed unnecessary second execution of ``xfconf-query``.
```

```suggestion
  - xfconf - removed unnecessary second execution of ``xfconf-query`` (https://github.com/ansible-collections/community.general/pull/1305).
```
Every entry needs a link.

---

Done in git push